includes/class-wc-gateway-ppec-client.php

```diff
@@ -811,23 +813,23 @@ protected function _get_paypal_line_items_from_order( $order ) {
 		$order    = wc_get_order( $order );
 
 		$items = array();
-		foreach ( $order->get_items( array( 'line_item', 'fee' ) ) as $cart_item_key => $values ) {
+		foreach ( $order->get_items( array( 'line_item', 'fee' ) ) as $cart_item_key => $order_item ) {
 
-
-			if( 'fee' === $values['type']) {
+			if( 'fee' === $order_item['type']) {
 				$item   = array(
-					'name'     => $values['name'],
+					'name'     => $order_item['name'],
 					'quantity' => 1,
-					'amount'   => round( $values['line_total'], $decimals),
+					'amount'   => round( $order_item['line_total'], $decimals ),
 				);
 			} else {
-				$amount = round( $values['line_subtotal'] / $values['qty'] , $decimals );
-				$item   = array(
-					'name'     => $values['name'],
-					'quantity' => $values['qty'],
+				$amount  = round( $order_item['line_subtotal'] / $order_item['qty'] , $decimals );
+				$product = version_compare( WC_VERSION, '3.0', '<' ) ? $order->get_product_from_item( $order_item ) : $order_item->get_product();
+				$item    = array(
+					'name'     => $order_item['name'],
+					'quantity' => $order_item['qty'],
 					'amount'   => $amount,
+					'sku'      => $product->get_sku(),
 				);
-
 			}
 
```

Comment:
@james-allan
LGTM 
There is an extra blank line here. Can you please remove it?

Reply:
Never mind, I did it myself.